client/src/components/atoms/style/Stepper.scss

```diff
@@ -0,0 +1,37 @@
+.stepper {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+
+    .step {
+        display: flex;
+        flex-direction: row;
+    }
+
+    .step-text {
+        margin-left: 10px;
+
+        h4 {
+            margin-bottom: 5px;
+            color: $faded-modal-text-color;
```

Comment:
minor colour difference: the colour in the figma design is `#8B8B8B` but `$faded-modal-text-color` is defined as `#7D7D7D`